cfme/tests/containers/test_containers_default_project_replicators.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import CheckboxTable
+from cfme.containers.replicator import Replicator
+from itertools import product
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.uncollectif(lambda provider: provider.version < 3.2),
```

Comment:
I noticed that this is not skipping the OSE 3.1 provider on PRT and I assume it's because the `uncollectif` marks are not stacking. What I'm trying to say is you will probably need to do `pytest.mark.uncollectif(lambda provider: current_version() < "5.6" or provider.version < 3.2)` instead of 2 separate `uncollectif` marks.